src/VisualStudio/Core/Def/Workspace/VisualStudioSymbolNavigationService.cs

```diff
@@ -142,11 +142,6 @@ internal partial class VisualStudioSymbolNavigationService : ForegroundThreadAff
 
                 var vsTextBuffer = (IVsTextBuffer)vsRunningDocumentTable4.GetDocumentData(documentCookie);
 
-                // Set the buffer to read only, just in case the file isn't
```

Comment:
`AbstractLanguageService` was already doing this (badly) so this was unnecessary